yarn.lock

```diff
@@ -0,0 +1,3261 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We already have package-lock, so we don't need this file (you can add to gitignore I guess)

Reply:
done